packages/swapper/src/swappers/zrx/zrxBuildTrade/zrxBuildTrade.ts

```diff
@@ -32,39 +31,20 @@ export async function zrxBuildTrade(
     wallet
   } = input
 
-  if (!sellAssetAccountId || !buyAssetAccountId) {
-    throw new SwapError(
-      'ZrxSwapper:ZrxBuildTrade Both sellAssetAccountId and buyAssetAccountId are required'
-    )
-  }
-
-  const buyToken = buyAsset.tokenId || buyAsset.symbol || buyAsset.network
-  const sellToken = sellAsset.tokenId || sellAsset.symbol || sellAsset.network
-  if (!buyToken) {
-    throw new SwapError(
-      'ZrxSwapper:ZrxBuildTrade One of buyAssetContract or buyAssetSymbol or buyAssetNetwork are required'
-    )
-  }
-  if (!sellToken) {
-    throw new SwapError(
-      'ZrxSwapper:ZrxBuildTrade One of sellAssetContract or sellAssetSymbol or sellAssetNetwork are required'
-    )
-  }
+  const buyToken = buyAsset.tokenId || buyAsset.symbol
```

Comment:
symbol is dangerous - add comment to default to a supported fee asset id for a given swapper in future